Automattic/wp-calypso
# Stats: Add new personal plan purchase page

Related to #81257

## Proposed Changes

* add a new page for purchasing personal plan (simplified view that doesn't use a purchase wizard component like the previous iterations) - based on changes introduced in #81250

## Testing Instructions



* navigate to the live branch
* **without any feature flag** applied verify that the experience didn't change for:
   * a page without any plan purchased (you should see the old wizard)
   * a page with the commercial plan purchased (you should see a confirmation notice)
   * a page with free or PWYW plan purchased (you should see the old wizard)
* apply `stats/type-detection` feature flag and check the updated experience
  * if the page **wasn't classified as either personal or commercial**, you should see the old wizard
  * if your page **has been classified as personal** you should see a new purchase page for only the personal plan (not the wizard) - this can be tested locally by forcing commercial detection in the controller (`isCommercial` option)
  * if you purchased free or PWYW plans you should see a confirmation full-page notice when visiting the purchase page again

<img width="1192" alt="SCR-20230904-kdoq" src="https://github.com/Automattic/wp-calypso/assets/112354940/7e0b40b0-6218-48b5-9e14-9516b3ea3025">


## Pre-merge Checklist
